client/src/helpers/APICalls/paymentService.ts

```diff
@@ -0,0 +1,25 @@
+import { PaymentMethod } from '../../interface/PaymentMethods';
+
+interface AddPaymentResponse {
+  clientSecret: string;
+  attachedDetails: {
+    name: string;
+    email: string;
+  };
+}
+export const createCardSetup = async (): Promise<AddPaymentResponse> => {
+  // create setup intent
```

Comment:
Remove comment

Reply:
@BerryGIT-ME you can actually remove the above comment